### DISCUSSION TITLE
Error: No Connected Devices!

### DISCUSSION THREAD

**Discussion by emmanueldlauron** [kitsuned/PotatoNV#90] [Q&A]
I installed the driver then it say's no connected devices.

![Screenshot 2021-08-21 190657](https://user-images.githubusercontent.com/41054824/130319837-91827e6a-324a-4954-9be2-c469addea77f.png)

**Comment by evileveline** [comment]
If anyone has the same problem - check the drivers, if the drivers are okay, try changing the jumper on the testpoint (for example, to a paper clip) - it helped me. The problem may be due to insufficient electrical voltage.

**Comment by actuallyjammy** [comment]
Same issue here.
